cmd/files/push/cmd_push.go

```diff
@@ -83,7 +88,7 @@ can be used to override detected file type.
 				Authorize:   authorize,
 				Directory:   directory,
 				FileType:    fileType,
-				Directives:  directives,
+				Directives:  mDirectives,
```

Comment:
Nitpick. The new name looks odd. Or is it common in Go to add a type to the name? I can propose `directivesMap`